packages/e2e-tests/src/specs/editor/authorUser.js

```diff
@@ -31,6 +31,9 @@ import {
 
 async function publishStory() {
   await expect(page).toClick('button', { text: 'Publish' });
+  // Bypass checklist
+  await expect(page).toClick('button', { text: 'Continue to publish' });
+  await page.waitForTimeout(1000);
```

Comment:
Why is this wait needed?

Reply:
The page timesout otherwise. i'll try it again without so you can see (or i'm proved wrong, which would be nice)